frontend/app/dashboard/page.tsx

```diff
@@ -153,6 +155,21 @@ export default function Dashboard() {
           </div>
         </div>
       </div>
+      {eventDataList.length === 0 && (
```

Comment:
can we do this as an either/ or with the event cards? ( ? : instead of &&) and can we also make it so that we avoid magic margins? could we just centre it in the page? would that work?